runtime/mandala/tests/integration_test.rs

```diff
@@ -1855,3 +1855,17 @@ fn currency_id_convert() {
 		);
 	});
 }
+
+#[test]
+fn parachain_subaccounts_are_unique() {
+	ExtBuilder::default().build().execute_with(|| {
+		let parachain: AccountId = ParachainInfo::parachain_id().into_account();
+		let relaychain = Utility::derivative_account_id(ParachainInfo::get().into_account(), 0);
+		let relaychain_subaccount_1 = Utility::derivative_account_id(ParachainInfo::get().into_account(), 1);
+		let relaychain_subaccount_2 = Utility::derivative_account_id(ParachainInfo::get().into_account(), 2);
+		assert_ne!(parachain, relaychain);
+		assert_ne!(parachain, relaychain_subaccount_1);
```

Comment:
no. I want to assert something like `relaychain_subaccount_1 == hex_literal!("12345678901234567890")`